src/frontend/magnify/src/components/rooms/RoomRow/RoomRow.tsx

```diff
@@ -123,6 +147,17 @@ export default function RoomRow({ room, baseJitsiUrl }: RoomRowProps) {
       ),
       onClick: openDeleteModal,
     });
+
+    result.push({
+      icon: <Copy />,
+      label: (
+        <Box alignSelf={'center'} margin={{ left: 'xsmall' }}>
+          {intl.formatMessage(messages.copyRoomLink)}
+        </Box>
+      ),
+      onClick: copyLinkToClipboard,
+    });
+
     return result;
   };
```

Comment:
Maybe merge the default actions into an array that you concatenate rather than push. (Pay attention to the order of actions, deleting is usually last for example and editing/settings first.)